robbot.py

```diff
@@ -76,10 +77,25 @@ async def get_cocktail(msg):
         print(ex)
         await msg.channel.send('Ayo, your code is wack.')
 
-# TODO Find replyed message through author
-# Make emebd 
-# Post to Pin
+'''
+@author: Keeth S.
+@dependencies: util/Pin.py
+@desc: Sends a embed to the Pin channel when a user reply's to a message with .pin
+@retunrs: async message back to channel confirming message was pinned
+# TODO Optimize Drink Object.
+'''
 async def pin_message(msg):
-    pass
+    try:
+        if not msg.reference:
+            await msg.channel.send('Sorry, bud. Just can\'t do it.')
```

Comment:
Should probably change the error message here to something more specific, e.g. "Which message should I pin? (reply to a message to pin it)"